src/datasource.ts

```diff
@@ -63,6 +85,8 @@ export class DataSource extends DatasourceWithAsyncBackend<AthenaQuery, AthenaDa
       table: this.templateSrv.replace(query.table),
     });
 
+  getWorkgroupEngineVersion = () => this.postResource('workgroupEngineVersion', { workgroup: this.workgroup });
```

Comment:
silly question maybe, but why is this a post resource instead of a get?

Reply:
mainly following the existing code where additional params are passed in i.e. `workgroup`.

not sure if this is the original reason for using `postResource` for these methods, but post requests don't encode the params in the url which is probably "safer". probably also easier to parse a request body instead of parsing a url to get the params on the backend.